M2/Macaulay2/m2/html.m2

```diff
@@ -120,20 +120,21 @@ html HTML := x -> demark(newline, {
 treatImgSrc := x -> apply(x, y -> if class y === Option and y#0 === "src" then "src" => toURL y#1 else y)
 html IMG := (lookup(html, IMG)) @@ treatImgSrc
 
-fixNewLines := method()
-fixNewLines Hypertext :=
-fixNewLines Nothing :=
-fixNewLines Option := identity
-fixNewLines Thing := x -> replace("\r\n","\n",toString x) -- toString prevents LaTeX being inserted...
+toStringMaybe := method()
```

Comment:
Personally I feel like `fixNewLines` was much more descriptive of what the method does. `toStringMaybe` doesn't say anything about literally the only change that this function makes!